index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Luis Laverde's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav class="navbar">
+    <div><img src="./imgs/my-logo/my-logo.png" /></div>
+    <div class="hamburguer-menu"><img src="./imgs/hamburguer-menu/hamburguer-menu-icon-enabled.png" /></div>
+  </nav>
+  <section class="name">
+    <header class="container">
```

Comment:
- Kindly the `header` tag used for your main header, not for sections so remove it here for sematic reasons 👍

Reply:
- Remove <header> tag used for main header for semantic reasons.
- Fix the indentation of all the <img>, <li> and <a> tags.
- Add alt attributes to the all img tags.